src/components/Post.tsx

```diff
@@ -12,16 +18,19 @@ import { MARKDOWN, PLAIN } from '../utils/constants';
 */
 
 type PostProps = {
-    id: number,
-    title: string, 
-    body: string, 
-    author: string, 
-    currentAuthor?: string
+    postId: string,
+    title: string,
+    body: string,
+    authorName: string,
+    authorId: number,
```

Comment:
the authorId in master is a string, not a number. The backend team changed this to string